src/documentation/pages/components/forms/select/examples/_sizes.vue

```diff
@@ -0,0 +1,86 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes" :tabs="exampleTabs">
+    p.-text(slot="example-description") Selects support the following sizes: <code>-xs</code>, <code>-sm</code>, <code>-md</code>, <code>-lg</code>, <code>-xl</code>. The default size is <code>-md</code>.
+    <Wrapper slot="example">
+      each size in ['xs', 'sm', 'md', 'lg', 'xl']
+        .-p--2
+          .chi-form__item(style="max-width:20rem")
+            label(class="chi-label" for=`example-size-${size}`) Label
+            select(class=`chi-select -${size}` id=`example-size-${size}`)
+              option=`-${size} Select`
+    </Wrapper>
```

Comment:
Portal does not support xl. We can create separate files for floating labels, error and sizes examples for portal
<img width="644" alt="Size" src="https://user-images.githubusercontent.com/92325462/149419398-6dacaddc-433e-411a-81a4-6da5f17490be.png">
.